src/model/transport.rs

```diff
@@ -162,6 +162,15 @@ pub struct VorgangTransportieren2010 {
     #[serde(skip)]
     #[cfg_attr(feature = "builder", builder(default))]
     _xmlns: ConstStr,
+    #[xml(
+        ns = b"xmlns",
+        name = b"ds",
+        ty = "attr",
+        value = "http://www.w3.org/2000/09/xmldsig#"
+    )]
+    #[serde(skip)]
+    #[cfg_attr(feature = "builder", builder(default))]
+    _xmlns_ds: ConstStr,
```

Comment:
@jueseitz I have problem here. If I try to deserialize XML without this attribute it fails. If I specify it as Option<ConstStr> - it is not adding `xmlns:ds` to root attributes.